addons/webinterface.default/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="webinterface.default"
-  version="2.4.5"
+  version="18.x-2.4.6"
```

Comment:
Any one knows if this (version format) will give issues?

Reply:
Not sure if it have issues, but more correct would be `2.4.6-18` (and `2.4.6-17` for Krypton).

Reply:
I was perhaps thinking 18.4.6 and 17.4.6 or something.
@jez500 ^^ Is this an option?

See https://github.com/xbmc/chorus2/pull/342 for discussion

Reply:
> Not sure if it have issues, but more correct would be 2.4.6-18 (and 2.4.6-17 for Krypton).

No, actually according to our current code the current format is more reasonable. It will parse `2.4.6` as revision and `18.x` as version. When comparing, it will first compare the version and then the revision, so it vaguely works like you would expect.